src/components/features/DeliveryGoodsInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Typography } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+
+interface GoodType {
+  title: string
+  value: string
+  quantity: number
+}
+
+export default function DeliveryGoodsInfo({ goodTypes }: any) {
+  const { t: translate } = useTranslation()
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5em', padding: '1em' }}>
+      <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+        <Typography>
```

Comment:
just use Typography prop for font weight.

Reply:
I know but all variants aren't appropriate. See this with h6. For me, it is very big.

![image](https://github.com/InterCarsBulgaria-TechLeadAcademy/warehouse.management.fe/assets/123353314/8b0dd43b-1d98-4082-88b6-d51a98767928)

Reply:
After the conversation, use fontWeight: 'bold'